mautic/mautic
# Froala code view editor - image CSS getting stripped

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | x
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Removing call to function in Froala image.js that removes many common style declarations and replaces them with a broken class.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create new email
2. Open builder
3. Insert image in text-slot
4. Open code view
5. Give image inline CSS: style="margin: 0; float: right; display: block; z-index: 1; position: relative; overflow: auto; vertical-align: top"
6. Close code view, then re-open
7. Observe that all inline CSS has been stripped from the image (very bad for emails).

#### Steps to test this PR:
1. Repeat steps 1-6 above
2. Observe that all inline CSS is no longer removed from the image
